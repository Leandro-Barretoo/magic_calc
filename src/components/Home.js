import React from 'react';

const Home = () => {
  const homeContainer = {
    margin: '3em',
    padding: '2em',
  };

  const paragraph = {
    margin: '3rem 0',
  };

  return (
    <div style={homeContainer}>
      <h2 style={{ fontSize: '2rem' }}>Welcome to our page!</h2>
      <p style={paragraph}>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        Phasellus elementum justo velit. Proin dui arcu, feugiat eget
        tellus vel, porta dapibus purus. Nam pulvinar metus sem,
        a ultricies ante volutpat ac. Donec ac massa ac metus lacinia
        molestie eget non erat. Curabitur ac turpis ut ex rutrum scelerisque.
        Proin tempus velit in magna aliquet fringilla.
        Aliquam mollis varius magna at placerat. Etiam laoreet luctus dui
        pharetra placerat.
        Sed sed neque mauris. Maecenas lorem nisi, pharetra in metus vel,
        tincidunt sagittis tellus.
        In et tempor risus. Nunc molestie tellus at sem accumsan,
        in vulputate tortor blandit.
        Sed non consectetur leo. Phasellus cursus consequat semper.
        Nunc feugiat consequat facilisis.
        Donec tempus vestibulum nisi, a blandit velit auctor nec.
      </p>
      <p style={paragraph}>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit.
        Phasellus elementum justo velit. Proin dui arcu, feugiat eget
        tellus vel, porta dapibus purus. Nam pulvinar metus sem,
        a ultricies ante volutpat ac. Donec ac massa ac metus lacinia
        molestie eget non erat. Curabitur ac turpis ut ex rutrum scelerisque.
        Proin tempus velit in magna aliquet fringilla.
        Aliquam mollis varius magna at placerat. Etiam laoreet luctus dui
        pharetra placerat.
        Sed sed neque mauris. Maecenas lorem nisi, pharetra in metus vel,
        tincidunt sagittis tellus.
        In et tempor risus. Nunc molestie tellus at sem accumsan,
        in vulputate tortor blandit.
        Sed non consectetur leo. Phasellus cursus consequat semper.
        Nunc feugiat consequat facilisis.
        Donec tempus vestibulum nisi, a blandit velit auctor nec.
      </p>
    </div>
  );
};

export default Home;
