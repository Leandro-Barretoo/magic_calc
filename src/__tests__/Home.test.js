import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';
import Home from '../components/Home';

describe('Testing the home component', () => {
  it('should render correctly to the dom', () => {
    const home = renderer.create(<Home />).toJSON();
    expect(home).toMatchSnapshot();
  });

  it('should render without errors', () => {
    render(<Home />);
  });
});
