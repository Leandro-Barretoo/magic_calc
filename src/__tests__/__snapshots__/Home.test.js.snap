// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the home component should render correctly to the dom 1`] = `
<div
  style={
    Object {
      "margin": "3em",
      "padding": "2em",
    }
  }
>
  <h2
    style={
      Object {
        "fontSize": "2rem",
      }
    }
  >
    Welcome to our page!
  </h2>
  <p
    style={
      Object {
        "margin": "3rem 0",
      }
    }
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus elementum justo velit. Proin dui arcu, feugiat eget tellus vel, porta dapibus purus. Nam pulvinar metus sem, a ultricies ante volutpat ac. Donec ac massa ac metus lacinia molestie eget non erat. Curabitur ac turpis ut ex rutrum scelerisque. Proin tempus velit in magna aliquet fringilla. Aliquam mollis varius magna at placerat. Etiam laoreet luctus dui pharetra placerat. Sed sed neque mauris. Maecenas lorem nisi, pharetra in metus vel, tincidunt sagittis tellus. In et tempor risus. Nunc molestie tellus at sem accumsan, in vulputate tortor blandit. Sed non consectetur leo. Phasellus cursus consequat semper. Nunc feugiat consequat facilisis. Donec tempus vestibulum nisi, a blandit velit auctor nec.
  </p>
  <p
    style={
      Object {
        "margin": "3rem 0",
      }
    }
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus elementum justo velit. Proin dui arcu, feugiat eget tellus vel, porta dapibus purus. Nam pulvinar metus sem, a ultricies ante volutpat ac. Donec ac massa ac metus lacinia molestie eget non erat. Curabitur ac turpis ut ex rutrum scelerisque. Proin tempus velit in magna aliquet fringilla. Aliquam mollis varius magna at placerat. Etiam laoreet luctus dui pharetra placerat. Sed sed neque mauris. Maecenas lorem nisi, pharetra in metus vel, tincidunt sagittis tellus. In et tempor risus. Nunc molestie tellus at sem accumsan, in vulputate tortor blandit. Sed non consectetur leo. Phasellus cursus consequat semper. Nunc feugiat consequat facilisis. Donec tempus vestibulum nisi, a blandit velit auctor nec.
  </p>
</div>
`;
